File: DayOne/Python/LevelThree/level_three_test.py
from unittest import TestCase

from level_three import *

class LevelThreeTest(TestCase):

    def test_that_i_have_a_2D_array_and_i_flatten_it_into_a_new_array(self):

        array = [[9, 0, 7], [3, 5, 1], [8, 1, 7], [9, 9, 6]]

        array_check = [9,0,7,3,5,1,8,1,7,9,9,6]

        self.assertEqual(get_flatten_array(array),array_check)

File: DayOne/Java/LevelThree/LevelThreeTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

public class LevelThreeTest{

    @Test
    public void testThatIhaveA2DArrayAndIFlattenItIntoANewArray(){

    int [][] array = {{9, 0, 7}, {3, 5, 1}, {8, 1, 7}, {9, 9, 6}};

    int [] arrayCheck = {9,0,7,3,5,1,8,1,7,9,9,6};

    int [] actual = LevelThree.getFlattenArray(array);

    assertArrayEquals(actual,arrayCheck);
    }
}
